
From: Hans Reiser <reiser@namesys.com>

This patch exports __pagevec_release and pagevec_lookup_tag.  As it looks like
they are intended to be external symbols.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/mm/swap.c |    3 +++
 1 files changed, 3 insertions(+)

diff -puN mm/swap.c~reiser4-export-pagevec-funcs mm/swap.c
--- 25/mm/swap.c~reiser4-export-pagevec-funcs	Wed Aug 18 16:42:42 2004
+++ 25-akpm/mm/swap.c	Wed Aug 18 16:42:42 2004
@@ -259,6 +259,8 @@ void __pagevec_release(struct pagevec *p
 	pagevec_reinit(pvec);
 }
 
+EXPORT_SYMBOL(__pagevec_release);
+
 /*
  * pagevec_release() for pages which are known to not be on the LRU
  *
@@ -388,6 +390,7 @@ unsigned pagevec_lookup_tag(struct pagev
 	return pagevec_count(pvec);
 }
 
+EXPORT_SYMBOL(pagevec_lookup_tag);
 
 #ifdef CONFIG_SMP
 /*
_
